bmaptools/BmapCopy.py

```diff
@@ -211,6 +212,28 @@ def __init__(self, image, dest, bmap=None, image_size=None):
 
         self._batch_blocks = self._batch_bytes // self.block_size
 
+    def set_psplash_pipe(self, pipe):
```

Comment:
Using a name like 'path' for the argument (instead of 'pipe') would improve readability a bit. Just 'pipe' makes me think it is an file-like object or some sort of pipe object, not a plain name.